docs/content/release_notes/whats_new.md

```diff
@@ -4,6 +4,28 @@ IBM® block storage CSI driver 1.11.0 introduces the enhancements that are detai
 
 **General availability date:** 15 January 2023
 
+## Enhanced dynamic host definition
```

Comment:
this is the name that we want to go out with? I thought it was only internal name

Reply:
@erantzabari will have to make the final decision on this. Unless we want it to have a different official name than the original host definition this section is to show what is new. it is enhanced and it is dynamic. there is no reason not to use this type of wording here.

Reply:
I agree with Matan here, the EHD (Enhanced host definition) is internal name. I would also not like to announce a big announcement around the host definition.
It is better to write on headline "Host Definition" and in the context - to remark it to "Enhancements for host definition were performed" or something like that..

Reply:
@erantzabari  @matancarmeli7  Here, I am not writing it as a name but as the fact that the existing host definition features are enhanced (added to and made better)
this is why I wrote like this:

```
Enhanced dynamic host definition

The IBM® block storage CSI driver 1.11.0 host definition feature now supports the following:
```

Reply:
@erantzabari  updated per our conversation. please approve.

Reply:
@matancarmeli7  please resolve this conversation. i had updated according to a conversation with eran and it will be reapproved later